src/wallet/rpc/wallet.cpp

```diff
@@ -235,7 +235,11 @@ static RPCHelpMan loadwallet()
                 },
                 RPCExamples{
                     HelpExampleCli("loadwallet", "\"test.dat\"")
+            + HelpExampleCli("loadwallet", "\"specialWallets/test2.dat\"")
+            + HelpExampleCli("loadwallet", "\"/Users/joe/specialWallets\"")
```

Comment:
`/Users/joe/specialWallets` should be changed to `/Users/joe/specialWallet/` here and in the RPC example since we only load one wallet at a time and to make clear that it is a directory.

Reply:
Thanks for the suggestion - fixed.